### PR TITLE
STONEBLD-28: Cleanups and fixes

### PR DESCRIPTION
**Pull request by Michkov**
- Pass repository secret via Secret instead of file on source workspace
- Merge configure-build task to init task
- Ensure that test tasks are working on private repositories

### CODE REVIEW

File: task/init/0.1/init.yaml [hunk 1]
```diff
@@ -19,19 +19,45 @@ spec:
...
+    - name: default-push-secret
+      csi:
+        readOnly: true
+        driver: csi.sharedresource.openshift.io
+        volumeAttributes:
+          sharedSecret: $(params.shared-secret)
   steps:
     - name: init
-      image: registry.access.redhat.com/ubi9/skopeo:9.1.0-5@sha256:6ebbdfce633f9915a6a99d8ee035d9fd258121aa1acf944229e90696eb353549
+      image: registry.redhat.io/openshift4/ose-tools-rhel8:v4.12
```
Comment (jsztuka) [inline task/init/0.1/init.yaml:38]: Digests are no longer requested?

**Review by jsztuka — APPROVED** [review]
lgtm

File: task/init/0.1/init.yaml (hunk 1 above)
Comment (Michkov) [reply task/init/0.1/init.yaml]: updated

**Review by tkdchen — APPROVED** [review]

**Review by dirgim — APPROVED** [review]
LGTM

**Pull request merged**